assets/reader-activation/auth.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * Internal dependencies.
+ */
+import './auth.scss';
+
+( function ( readerActivation ) {
+	window.onload = function () {
+		if ( ! readerActivation ) {
+			return;
+		}
+
+		const container = document.querySelector( '#newspack-reader-activation-auth-form' );
+		if ( ! container ) {
+			return;
+		}
+
+		const initialForm = container.querySelector( 'form' );
+		let form;
+		/** Workaround AMP's enforced XHR strategy. */
+		if ( initialForm.getAttribute( 'action-xhr' ) ) {
+			initialForm.removeAttribute( 'action-xhr' );
+			form = initialForm.cloneNode( true );
+			initialForm.replaceWith( form );
+		} else {
+			form = initialForm;
+		}
+
+		const actionInput = form.querySelector( 'input[name="action"]' );
+		const emailInput = form.querySelector( 'input[name="email"]' );
+		const passwordInput = form.querySelector( 'input[name="password"]' );
+		const submitButton = form.querySelectorAll( '[type="submit"]' );
```

Comment:
For clarity:

```suggestion
		const submitButtons = form.querySelectorAll( '[type="submit"]' );
```

Reply:
Fixed in e93e5283be032a86c4b7981c580861b00efb6f0d